homeassistant/components/config/device_registry.py

```diff
@@ -19,6 +19,7 @@
     vol.Required('type'): WS_TYPE_UPDATE,
     vol.Required('device_id'): str,
     vol.Optional('area_id'): vol.Any(str, None),
+    vol.Optional('friendly_name'): vol.Any(str, None),
```

Comment:
I think `friendly_name` might be a bit ambiguous, what about `name_by_user`?

Reply:
sure, i'll change it

Reply:
done